src/applications/disability-benefits/all-claims/tests/pages/contactInformation.unit.spec.jsx

```diff
@@ -12,29 +15,49 @@ import {
   STATE_VALUES,
   MILITARY_STATE_VALUES,
 } from 'applications/disability-benefits/all-claims/constants';
+import { commonReducer } from 'platform/startup/store';
+import reducers from '../../reducers';
 
 // const NEXT_YEAR = moment()
 //   .add(1, 'year')
 //   .format('YYYY-MM-DD');
 
+/* eslint-disable camelcase */
```

Comment:
Disabled camelcase linting rule because the feature toggle names are snackcase, and we will be removing these feature toggles in the very near future.